src/lib/Scenes/Collection/__tests__/__snapshots__/Collection-tests.tsx.snap

```diff
@@ -779,13 +795,13 @@ exports[`Collection renders a snapshot 1`] = `
                         "plural": false,
                         "type": "Image",
                       },
-                      "marketingCollection.artworksConnection.merchandisableArtists.image.resized": Object {
+                      "marketingCollection.artworksConnection.merchandisableArtists.image.url": Object {
                         "enumValues": null,
                         "nullable": true,
                         "plural": false,
-                        "type": "ResizedImageUrl",
+                        "type": "String",
                       },
-                      "marketingCollection.artworksConnection.merchandisableArtists.image.resized.url": Object {
+                      "marketingCollection.artworksConnection.merchandisableArtists.initials": Object {
```

Comment:
Is `initials` a field we are getting back from the graphql response? I didn't think it was in which case I'm confused about how its being used in the component.

Reply:
It looks like it is! https://github.com/artsy/emission/blob/master/src/lib/Components/ArtistListItem.tsx#L161

Reply:
This is used inside of the[ EntityHeader](https://github.com/artsy/palette/blob/f2b266d289dae2c387099f8e4acc842be3147e00/packages/palette/src/elements/EntityHeader/EntityHeader.ios.tsx#L36-L40) component. If there isn't an image for the EntityHeader it falls back to use the initials.